CHANGELOG.md

```diff
@@ -18,7 +18,7 @@
 
 ## New Functionality
 
--
+- [openapi-generator] Added a feature to use a config.json with `-c/--config` to use the openapi-generator. The config.json can be used for all generator options that would otherwise be used with command line options. If command line options are used, they overwrite the options set in the config file.
```

Comment:
Sorry, I think I wasn't clear enough.
I think in the release notes you don't need too much detail, try to be concise but explicit. I think "config.json" is not 100% correct.
The note should always be imperative ("add" instead of "added").
```suggestion
- [openapi-generator] Add the `-c/--config` command line option to specify options through a configuration file instead of on the command line.
```